dom/practice/js/scripts.js

```diff
@@ -0,0 +1,39 @@
+// 1: Set the text of the <h1> element
+const headline = document.querySelector('h1');
+headline.textContent = `My Activities List`;
+
+// 2: Set the color of the <h1> to a different color
+headline.style.color = 'skyblue';
+
+// 3: Set the content of the '.desc' paragraph
+// The content should include at least one HTML tag
```

Comment:
You can do multi line comments like this:
```
/*
line 1
line 2
*/
```

Reply:
That's funny, I caught that too and was surprised the instructor didn't do that.

Reply:
It depends but I find myself doing multiple single line comments as well.
It just looks better to me and a little more natural but it doesn't matter.
Whatever works better for you.